src/Compilers/CSharp/Portable/Binder/UsingStatementBinder.cs

```diff
@@ -132,9 +144,33 @@ internal override BoundStatement BindUsingStatementParts(DiagnosticBag diagnosti
                 expressionOpt,
                 iDisposableConversion,
                 boundBody,
+                disposeMethod,
                 hasErrors);
         }
 
+        /// <summary>
+        /// Checks for a Dispose method on exprType. Failing to satisfy the pattern is not an error -
+        /// it just means we have to check for an interface instead.
```

Comment:
I think this comment is wrong now since we check for an interface before looking for the pattern. I don't think that this code should add any errors, though. The error reporting in the other methods makes sense.